comms/Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM golang:alpine AS builder
 
+ARG GOOS=linux
```

Comment:
Changes to this file are dangerous and brick linux

Reply:
reverting, thanks